docs/en/deployment-guides/horizontal-scaling.md

```diff
@@ -0,0 +1,468 @@
+---
+slug: /en/architecture/horizontal-scaling
+sidebar_label: Scaling out
+sidebar_position: 10
+title: Scaling out
+---
+import ReplicationShardingTerminology from '@site/docs/en/_snippets/_replication-sharding-terminology.md';
+import ConfigFileNote from '@site/docs/en/_snippets/_config-files.md';
+
+
+## Description
+This example architecture is designed to provide scalability.  It includes three nodes: two combined ClickHouse plus coordination (ClickHouse Keeper) servers, and a third server with only ClickHouse Keeper to finish the quorum of three. With this example, we'll create a database, table, and a distributed table that will be able to query the data on both of the nodes.
```

Comment:
I have to come up with a different term instead of standalone, as in this example to make it super simple the third node is using the ClickHouse server install and not the standalone Keeper.